newsubprojects.md

```diff
@@ -0,0 +1,183 @@
+
+If a Subproject team knows from the start that it would eventually like its Subproject to be part
+of the official Jupyter organization, it can be incubated in the
+[jupyter-incubator](https://github.com/jupyter-incubator) organization.
+
+The goals of the incubation in the [jupyter-incubator](https://github.com/jupyter-incubator) organization are as follows:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while
+  complying with individual and organizational contribution restrictions.
+* Contributors can work with the community and gather feedback that will help them develop and
+  refine a clear and concise integration proposal.
+* Contributors can get feedback early and often from the Steering Council and community.
+* Enables the core development team to assess the readiness and applicability of proposed
+  technologies against a backdrop of the broader project design efforts.
+* Allow the community and Steering Council to re-evaluate existing Subprojects in the main
+  jupyter GitHub organizations to determine if any need to be moved to jupyter-incubator.
+* Allow the community to make sure that Subprojects develop into something that is
+  appropriate to become an officially supported and maintained part of the project.
+
+### Proposal for incubation
+
+A Subproject team can initiate the [jupyter-incubator](https://github.com/jupyter-incubator)
+incubation process by submitting a proposal for incubation. This process is designed to be
+lightweight and quick:
+
+1. The proposers must fill out incubation application by sumitting a pull request to the
+   [jupyter-incubator/proposals](https://github.com/jupyter-incubator/proposals) repository.
+2. Proposers must announce their intent to the community with a post to the main Jupyter
+   list.
+2. The proposed Subproject must have an Advocate who is an active Steering Council member.
+3. The proposal for incubation will be discussed by the community and approved or rejected by
+   the consensus of the Steering Council.
+4. If approved, the proposal pull request will be merged, if rejected, it will be closed.
+
+### Incubation period
+
+Once a Subproject is approved for incubation in the
+[jupyter-incubator](https://github.com/jupyter-incubator) organization, the following steps will be
+taken:
+
+
+1. A GitHub repo will be created under the jupyter-incubator organization
+2. A GitHub team will be created with read/write access to the repo, including the
+   Advocate.
+3. The standard Jupyter LICENSE file will be added to the repository.
+4. A member of the Steering Council will announce the newly incubated Subproject on the
+   main Jupyter list.
+
+The goal of the incubation period is to demonstrate that the Subproject is going to develop an
+with an active developer and user community. There is no particular length of time required,
+but most projects will be in incubation for at least 6 months to 1 year.
```

Comment:
"There is no particular length of time required for incubation, but **it is expected that** most projects will be incubation for at least 6 months to a year."